artifacts/CAMARA_common.yaml

```diff
@@ -265,6 +265,16 @@ components:
       status: 501
       code: NOT_IMPLEMENTED
       message: This functionality is not implemented yet
+  NotSupported:
+   description: Not Supported
+   content:
+    application/json:
+     schema:
+      $ref: "#/components/schemas/ErrorInfo"
+     example:
+      status: 501

```

Comment:
To be aligned with Issue#127 output (so far 422)

Reply:
@PedroDiez 
Understand there is a difference between 4xx and 5xx HTTP error codes.
> The 4xx codes are intended for cases in which the client has erred, and the 5xx codes for the cases in which the server is aware that the server has erred.

In this context, the client cannot "correct" the request as the server is not supporting this specific request.

Reply:
@mhfoo please see my comment https://github.com/camaraproject/Commonalities/issues/127#issuecomment-2094346214 regarding your same statement there. 422 with appropriate description is definitely more helpful for the client then 501.

Reply:
I think 501 is appropriate for functionality that has not _yet_ been implemented, but will be (or, at least, could be) implemented at some point in the future, whereas a 4XX error should be returned for a request that will never make sense and will never be supported (such as requesting SIM Swap status for a landline).

However desirable it is that all implementations support all possible functionality from the start, I do not think that will be the case.

So I support introducing a 501 error, but we need to be careful how it is used.